### PR TITLE
check if mesos_ip is defined

### PR DESCRIPTION
**Pull request by tsaridas**
check if mesos_ip is defined else listen to all interfaces.

### CODE REVIEW

**Comment by bbossy** [comment]
LGTM

**Comment by bbossy** [comment]
wait...

File: templates/conf-mesos.j2 [hunk 1]
```diff
@@ -3,7 +3,9 @@
 # special options supported by
 # mesos-init-wrapper
 CLUSTER="{{mesos_cluster_name}}"
+{%- if mesos_ip is defined %}
```
Comment (froesef-zz) [inline templates/conf-mesos.j2:6]: so if there is no IP set in the mesos config file it listens on all interfaces?

**Review by bbossy — APPROVED** [review]

**Comment by tsaridas** [comment]
Hi @bbossy !
This was not needed after all that is why its not merged :)

**Comment by bbossy** [comment]
Hi @tsaridas !
Thanks for the information! Closing then.

Greetings!

**Pull request closed without merging**